client/packages/common/src/hooks/useNativeClient/useNativeClient.ts

```diff
@@ -126,11 +170,15 @@ export const useNativeClient = ({
     if (!nativeAPI) return;
     if (!autoconnect) return;
     if (!previousServer) return;
+    if (mode !== NativeMode.Client) return;
```

Comment:
does this mean if `Server` mode was picked, it wouldn't also work as client ?

Reply:
I was catering for a pretty specific scenario, which, yeah, affects the autoconnect. 
Will just set the previousServer when selecting server